src/utils/colonyClient.ts

```diff
@@ -19,3 +21,20 @@ export const getCachedColonyClient = async (
   colonyClientsCache[colonyAddress] = colonyClient;
   return colonyClient;
 };
+
+export const getColonyFromDB = async (
```

Comment:
We fetch the colony from the db here (only a very limited set of fields currently), so we can get the set of tokens that belong to the colony. This is needed in the event we need to remove tokens.

It does not appear to be the case that this can be queried directly (e.g. "getColonyTokensByColonyId"), since we're using a many-to-many relationship to join them. https://docs.amplify.aws/cli/graphql/data-modeling/#many-to-many-relationship